packages/building-webpack/default-config.js

```diff
@@ -0,0 +1,130 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const { BabelMultiTargetPlugin } = require('webpack-babel-multi-target-plugin');
+const BabelPluginHTMLMinifier = require('babel-plugin-template-html-minifier');
+const CleanWebpackPlugin = require('clean-webpack-plugin');
+
+const ENV = process.argv.find(arg => arg.includes('production')) ? 'production' : 'development';
+const developmentEs5 = !!process.argv.find(arg => arg.includes('es5'));
+
+const defaultOptions = {
+  indexHTML: './index.html',
+  indexJS: ['./index.js'],
+};
+
+const es6Target = {
+  browsers: [
+    'last 2 Chrome major versions',
+    'last 2 ChromeAndroid major versions',
+    'last 2 Edge major versions',
+    'last 2 Firefox major versions',
+    'last 3 Safari major versions',
+    'last 3 iOS major versions',
+  ],
+  tagAssetsWithKey: false,
+  esModule: true,
+};
+
+const es5Target = {
+  browsers: ['ie 11'],
+  tagAssetsWithKey: true,
+  noModule: false,
+};
+
+function getBrowserTargets() {
+  // production: es6 and es5 output
+  if (ENV === 'production') {
+    return { es6: es6Target, es5: es5Target };
+  }
+
+  // development: es5 output for testing IE11
+  if (developmentEs5) {
+    return { es5: es5Target };
+  }
+
+  // development: only es6 output for speed
+  return { es6: es6Target };
+}
+
+// eslint-disable-next-line
+module.exports = userOptions => {
+  const options = {
+    ...defaultOptions,
+    ...userOptions,
+  };
+
+  const config = {
+    entry: [options.indexJS],
+
+    output: {
+      filename: '[name].[chunkhash].js',
+      chunkFilename: '[name].[chunkhash].js',
+      path: path.resolve(process.cwd(), 'dist'),
+    },
+
+    resolve: {
+      mainFields: ['es2015', 'module', 'main'],
+    },
+
+    module: {
+      rules: [
+        {
+          test: /\.js$/,
```

Comment:
`npm run build` only works if I add here
```
 exclude: [modulePath => {
   return (
     /node_modules/.test(modulePath) &&
     !/node_modules\/lit-html/.test(modulePath) &&
     !/node_modules\/@open-wc/.test(modulePath)
   );
 }],
```

probably something to do with too many files to process or so :/
=> I guess having a static include/exclude list is still needed  
=> and to honest even if the auto magic from BabelMultiTargetPlugin would work I don't understand why this plugin would tackle that totally unrelated problem

Reply:
It's not an unrelated problem. In order for the differential serving to work, the es2015 output should contain es2015 code while the es5 output should contain es5 code. Therefore there needs to be come logic to determine what should be transpiled and what shouldn't.

This project is not really large at all, the dependencies are the web component polyfills, lit-html and lit-element. In your exclude you're not compiling lit-element and the polyfills so probably it won't work on IE11. Maybe it's one of the polyfills that's causing issues for you>

Reply:
it's an include list and we could have it in the default config?

e.g. something like this
```
defaultConfig({ babelInclude: ['lit-element', 'lit-html', '@open-wc'] })
```

I guess now it's a problem either with the BabelMultiTargetPlugin => think's it needs to transpiles all of npm and then it "breaks"
or with npm itself e.g. it can't process that many files on windows10

in either case I would not like to have a config that only works on mac/linux :/ as i could not use it myself then :(

if this needs a workaround until it's fixed somewhere - I would say so be it... 

what do you think?

Reply:
If the issue is the amount of files, then this setup won't work at all. Because as soon as you add a few more dependencies or develop some code then you will reach this point. There isn't a lot of code in there right now at all.

Let's check if someone with windows has the same issue.

Reply:
oh boy it really is lit-element 😭 
if I do
````
 exclude: [modulePath => {
   return (
     /node_modules/.test(modulePath) &&
     !/node_modules\/lit-html/.test(modulePath) &&
     !/node_modules\/lit-element/.test(modulePath) &&
     !/node_modules\/@open-wc/.test(modulePath)
   );
 }],
```

it again stops working... :/

Reply:
ok that is a window10 WSL problem 😭 
works fine in my plain bash... 😒